### PR TITLE
Setting up for your first day...

### PR DESCRIPTION
**Pull request by al-the-x**
- [x] **NEW** Prep-work Assignment: _Ready Player One_
- [x] **NEW** WIKI ALL THE THINGS!
- [x] **NEW** Glorious new README!

### CODE REVIEW

File: 00--Ready-Player-One/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   * [ ] `hub`
+   * [ ] `node`
+  * [ ] Add the output of the following commands as comments on your *WIP Issue*:
+    * [ ] `which zsh && zsh --version`
+    * [ ] `which git && git --version`
+    * [ ] `which hub && hub --version`
+    * [ ] `which node && node --version`
+  * [ ] Install the following (bonus if you use `brew`):
+    * [ ] [Google Chrome](http://google.com/chrome)
+    * [ ] [Atom](http://atom.io)
```
Comment (al-the-x) [inline 00--Ready-Player-One/README.md:38]: Add Dash and Slack back in...

**Comment by al-the-x** [comment]
Some questions from Chris's survey that we might consider including:

> What difficulties do you predict that you will encounter during our course? How have you reacted to similar situations in the past? How do you plan on reacting to these situations during the class?
> 
> Which part of the prework did you like the MOST? Which part of the prework did you like the LEAST? If you found any of the prework confusing, please explain.

Maybe make an essay out of this? I can totally see the value of a guided writing assignment here, e.g. bright bulb, dim bulb, blocker on prep work.

> What is program Control Flow? 
> What is the difference between an array and a hash?

I can see rolling these up into a larger part of the assignment, as well, e.g. "Describe the following terms, based on your research. If you don't know how to define them or you feel a little fuzzy, do some more research. Attribute your sources by including links."

> Write a set of command line commands which accomplish the following (in this order): 
>   (a) change directory to your home directory; 
>   (b) look at the contents of your home directory; 
>   (c) make a new directory called "iron_yard" inside your home directory; 
>   (d) copy the file called ".profile" into the new "iron_yard" directory; 
>   (e) change into that directory; 
>   (f) remove the copy of the ".profile" file within "iron_yard".

Trickier to test. I'd almost rather give them an assignment that includes pushing code into Github... Kind of like the "create your `TIY-Assignments` repo" piece that we took out.

> Write a CSS selector to find all `<span>` HTML elements with a class of "important" which happen to be inside of any `<h3>` elements.

This could easily be an assignment in CodePen that they fork and link to their WIP Issue. What objectives would that test?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Throughout the course, I'll be adding files to this repository to distribute assignments, notes for each class, and general information about the course. You'll open, close, and contribute to Issues and Pull Requests ([more about "issues" on Github](https://help.github.com/articles/about-issues)) in this repo and others that you'll create to turn in assignments and show your work.  By the end, you'll all be Github experts!
+
+I know there's not a lot of time before class starts; Kelly and I are working hard to get everything ready for you. I can't wait...
+
+###### David | http://al-the-x.me
+
+* Github: [@al-the-x](https://github.com/al-the-x)
+* Twitter: [@al_the_x](https://twitter.com/al_the_x)
+* LinkedIn: [althex](https://linkedin.com/in/althex)
```
Comment (kellymurray) [inline README.md]: ###### Kelly

* GitHub: [@kellymurray](https://github.com/kellymurray)
* Twitter: [@kellyemm](https://twitter.com/kellyemm)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+## Hello and welcome to Front End Engineering!
+
+Hi, everyone! I'm David Rogers, and I'm the new Front-End Instructor for the Durham campus. I've been teaching FEE for The Iron Yard since September 2014, when I helped launch the campus in Orlando, and now my family and I are moving to Durham so I can teach here.
+
+Kelly and I are really looking forward to meeting each of you in person and getting to know you better over the course of the upcoming 12 weeks. We're also really excited about teaching and mentoring you all, seeing what awesome things you'll build and what interesting ways you'll approach the assignments and projects in the class.
+
+First, you need to [sign up for a free Github account](http://github.com/signup), find [the class project](https://github.com/TIY-Durham/2015-FALL-FEE) and "star" it ([more on "starring" and notifications](https://help.github.com/articles/about-stars)). Once you do, I'll invite you to [a group for our class](https://github.com/orgs/TIY-Durham/teams/2015-fall-fee), so watch for an email from Github!
```
Comment (kellymurray) [inline README.md]: > ### Please complete the prep-work below _before class on Monday_...
> 
> You should have received a pre-class assignment from Chris, and it's important that you **finish that work before class on Monday.** The assignment that I give isn't significantly different, _but the turn-in instructions are._ They're designed to get you ready for the in-class routine and how _all_ your assignments will be submitted. _Please take the time to complete it!_

**Pull request merged**